Dockerfile

```diff
@@ -1,17 +1,15 @@
 # Base container that is used for both building and running the app
-FROM fedora:30 as base
-ARG RUBY_MODULE="ruby:2.6"
-ARG NODEJS_MODULE="nodejs:11"
+FROM registry.fedoraproject.org/fedora-minimal:30 as base
+ARG RUBY_VERSION="2.6"
+ARG NODEJS_VERSION="11"
 
 RUN \
-  echo "tsflags=nodocs" >> /etc/dnf/dnf.conf && \
-#  dnf -y upgrade && \
-  dnf -y module install ${RUBY_MODULE} ${NODEJS_MODULE} && \
-  dnf -y install mysql-libs mariadb-connector-c postgresql-libs ruby{,gems} rubygem-{rake,bundler} nc hostname \
+  echo -e "[nodejs]\nname=nodejs\nstream=${NODEJS_VERSION}\nprofiles=\nstate=enabled\n" > /etc/dnf/modules.d/nodejs.module && \
```

Comment:
Not that I know exactly, but isn't there a DNF command to do this? Normally it's `dnf module enable nodejs:${NODEJS_VERSION}` or even `dnf module install ...`. Does microdnf provide a similar feature?

Reply:
sadly no, see https://fedoramagazine.org/building-smaller-container-images/

Reply:
or the actual source code https://github.com/rpm-software-management/microdnf/tree/master/dnf/plugins

Reply:
I expected as much. Guess we now know why it's micro ;)

Reply:
@ekohl the sad part is that at the end, our image include ruby, node and python :( so much for micro....